### PR TITLE
DOCS-1445: Update data management landing page

### PR DESCRIPTION
**Pull request by sguequierre**
* Integrate proposal from @MirandaKapin to update wording on this page 

Diagram to be added

### CODE REVIEW

File: docs/data/_index.md [hunk 1]
```diff
@@ -19,23 +19,33 @@ menuindent: true
...
 
 ![Data is captured on the machine, uploaded to the cloud, and then deleted off local storage.](/data/data_management.png)
 
-The service has two parts: [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/).
+Viam's data management features can be broken down into on-device and cloud data capabilities:
 
-Once you have captured and synced data, you can:
+- **On-Device Data Management:** Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+  - **[Data Capture](/data/capture/):** Configurable for various robot components like cameras, sensors, and motors; set capture frequency, with automatic resumption after robot restarts.
+  - **[Cloud Sync](/data/cloud-sync/):** Securely transfers data to cloud at user-defined frequency, resilient to interruptions, and deletes local data post-sync for space management.
```
Comment (npentrel) [inline docs/data/_index.md:35]: I think this looks a bit cramped. How would you feel about cards instead? (For this and the cloud data management section)
```suggestion
### On-Device Data Management

Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.

{{< cards >}}
{{% manualcard link="/data/capture/" %}}

#### Data Capture

Configurable for various robot components like cameras, sensors, and motors; set capture frequency, with automatic resumption after robot restarts.

{{% /manualcard %}}
{{% manualcard link="/data/cloud-sync/" %}}

#### Cloud Sync

Securely transfers data to cloud at user-defined frequency, resilient to interruptions, and deletes local data post-sync for space management.

{{% /manualcard %}}
{{< /cards >}}
```

**Review by npentrel — COMMENTED** [review]
Looks good to me, main question is whether we want to use cards instead

File: docs/data/_index.md (hunk 1 above)
Comment (sguequierre) [reply docs/data/_index.md:35]: Great suggestion! Will implement.

File: docs/data/_index.md [hunk 2]
```diff
@@ -19,23 +19,82 @@ menuindent: true
...
+
+Securely transfer data to cloud at the frequency you define, resilient to interruptions, and delete local data post-sync for space management.
+
+{{% /manualcard %}}
 {{< /cards >}}
 
-## Used with
+### Cloud data management
+
+Streamlined data handling with advanced querying, viewing, and filtering capabilities, along with efficient data labeling and exporting tools.
```
Comment (sguequierre) [inline docs/data/_index.md:56]: TODO (comment got outdated): reframe descriptions here to speak to user more than just highlight capabilities

**Review by npentrel — APPROVED** [review]

**Review by MirandaKapin — CHANGES_REQUESTED** [review]
~~Then, you can configure which data to sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine.
The data management service also keeps your data secure during transfer.~~

Once collected, you can configure which data to securely sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine.

**Comment by sguequierre** [comment]
> ~Then, you can configure which data to sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine. The data management service also keeps your data secure during transfer.~
> 
> Once collected, you can configure which data to securely sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine.

@MirandaKapin Done!

**Review by MirandaKapin — APPROVED** [review]
Looks good!

**Pull request merged**